src/v0/destinations/the_trade_desk/networkHandler.js

```diff
@@ -0,0 +1,86 @@
+const { NetworkError, AbortedError } = require('@rudderstack/integrations-lib');
+const { httpSend, prepareProxyRequest } = require('../../../adapters/network');
+const {
+  processAxiosResponse,
+  getDynamicErrorType,
+} = require('../../../adapters/utils/networkUtils');
+const { getSignatureHeader } = require('../../../cdk/v2/destinations/the_trade_desk/utils');
+const { isHttpStatusSuccess } = require('../../util/index');
+const tags = require('../../util/tags');
+const { JSON_MIME_TYPE } = require('../../util/constant');
+
+const proxyRequest = async (request) => {
+  const { endpoint, data, method, params, headers, config } = prepareProxyRequest(request);
+
+  if (!config?.advertiserSecretKey) {
+    throw new AbortedError('Advertiser secret key is missing in destination config. Aborting');
+  }
+
+  if (!process.env.THE_TRADE_DESK_DATA_PROVIDER_SECRET_KEY) {
+    throw new AbortedError('Data provider secret key is missing. Aborting');
```

Comment:
These should be Instrumentation Errors as customer misconfigured the destination

Reply:
it is coming from environment variable.

Reply:
plaformError is the correct class we can use here.